packages/frontend/src/components/SimpleStatistic/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import {
+    BigNumber,
+    BigNumberContainer,
+    BigNumberLabel,
+    SimpleStatisticsWrapper,
+} from './SimpleStatistics';
```

Comment:
Have we stopped using `.styles.tsx` in the name ?

Reply:
Good catch! completely missed that! 👀